addons/search-sprites/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Search sprites by name",
+  "name": "Search box in sprite pane",
```

```suggestion
  "name": "Sprite pane search",
```
The current name is fine too, I just like shorter ones.

---

Personally I think "search box" is OK as it makes it obvious that this adds a new search bar.